pkg/vcs/linux.go

```diff
@@ -176,11 +176,25 @@ func linuxDisableConfigs(config []byte, tags map[string]bool) []byte {
 		// which makes bisections take weeks.
 		"CONFIG_DEBUG_KOBJECT": "disable-always",
 	}
-	for cfg, tag := range prereq {
+	for cfg, tag := range disable {
 		if !tags[tag] {
 			config = bytes.Replace(config, []byte(cfg+"=y"), []byte("# "+cfg+" is not set"), -1)
 		}
 	}
+	alter := []struct {
+		From string
+		To   string
+		Tag  string
+	}{
+		// Even though ORC unwinder was introduced a long time ago, it might have been broken for
+		// some time. 5.4 is chosen as a version tag, where ORC unwinder seems to work properly.
+		{"CONFIG_UNWINDER_ORC", "CONFIG_UNWINDER_FRAME_POINTER", "v5.4"},
+	}
+	for _, a := range alter {
+		if !tags[a.Tag] {
+			config = bytes.Replace(config, []byte(a.From+"=y"), []byte(a.To+"=y"), -1)
```

Comment:
But the config also contains:
# CONFIG_UNWINDER_FRAME_POINTER is not set

so after the replacement we will end up with:
CONFIG_UNWINDER_FRAME_POINTER=y
# CONFIG_UNWINDER_FRAME_POINTER is not set

How does olddefconfig react to this? Won't it be still disabled?
Note that for some future config that we will want to alter we may also end up with:
# FOO is not set
FOO=y

I assume at least 1 of these combinations won't work. Most likely when somebody will add that FOO in future, they will not test (assuming this functionality already works), so it will silently break.

We may need something more robust here.

Reply:
Currently this works as we want, so `FRAME_POINTER` ends up being enabled. But I've switched to explicitly enabling it and disabling `ORC` it just in case, PTAL.